packages/tpetra/core/example/Lesson02-Map-Vector/lesson02_read_modify_vec_kokkos.cpp

```diff
@@ -50,6 +50,7 @@
 #include <Tpetra_Core.hpp>
 #include <Tpetra_Vector.hpp>
 #include <Tpetra_Version.hpp>
+#include <Teuchos_CommHelpers.hpp>
```

Comment:
Same as above

Reply:
See above comment.  The goal is to remove unnecessary includes from `Tpetra_ConfigDefs.hpp`.  I didn't want to do that in this PR quite yet, because it will almost certainly break downstream apps (I've encountered that issue with Albany, for example, when I tried removing some includes from `Teuchos_ConfigDefs.hpp`).